README.md

```diff
@@ -39,98 +41,165 @@ Current stable release is able to persist Orion context data in:
+
+    ngsi-event={
+        http-headers={
+            Content-Length: 492
+            User-Agent: orion/0.9.0
+            Host: localhost:1028
+            Accept: application/xml, application/json
+            Content-Type: application/json
+            Fiware-Service: vehicles
+            Fiware-ServicePath: 4wheels 
+        },
+        payload={
+            {
+                "subscriptionId" : "51c0ac9ed714fb3b37d7d5a8",
+                "originator" : "localhost",
+                "contextResponses" : [
+                    {
+                        "contextElement" : {
+                        "attributes" : [
+                            {
+                                "name" : "speed",
+                                "type" : "kmh",
+                                "value" : "112.9",
+                                "metadatas": []
+                            },
+                            {
+                                "name" : "oil_level",
+                                "type" : "percentage",
+                                "value" : "74.6",
+                                "metadatas": []
+                            }
+                        ],
+                        "type" : "car",
+                        "isPattern" : "false",
+                        "id" : "car1"
+                    },
+                    "statusCode" : {
+                        "code" : "200",
+                        "reasonPhrase" : "OK"
+                    }
+                ]
+            }
+        }
+    }
 
-    event={
-		body=json_data,
-		headers={
-			content-type=application/json,
-			fiware-service=my_company_name,
-			fiware-servicepath=workingrooms_floor4,
-			timestamp=1402409899391,
-			transactionId=asdfasdfsdfa,
-			ttl=10,
-			destination=Room1-Room
-		}
-	}
+[Top](#top)
 
-<b>NOTE: The above is an <i>object representation</i>, not Json data nor any other data format.</b>
+###<a name="section3.3"></a>From NGSI format to Flume format
+Flume events are not much more different than the above representation: there is a set of headers and a body. This is an advantage, since allows for a quick translation between formats. Thus, once the notification is received, the `HTTPSource` relies on the custom `OrionRestHandler` for checking its validity (that it is a POST request, that the target is 'notify' and that the headers are OK), detecting the content type (in the example, Json format), extracting the data (the Json part) and finally creating the Flume event to be put in the native `MemoryChannel`.
+
+The equivalent <i>object representation</i> (not any real data format) for such a notified NGSI event could be the following Flume event:
+
+    flume-event={
+        headers={
+	         content-type=application/json,
+	         fiware-service=vehicles,
+	         fiware-servicepath=4wheels,
```

Comment:
Maybe is worth commenting it (i.e. that the Flume header fiware-servicepath is a post-processed version of the Fiware-ServicePath HTTP header comming from the NGSI-sources, which scaping rules are described at [link to the right .md where it is explained] along thiw example?